### ISSUE TITLE
Hyperparameter tuning for num_leaves and learning rate

### ISSUE THREAD

**Issue by camroach87** [camroach87/pod-energy-comp#12]
The num_iterations and learning_rate arguments are very important according to the [documentation](https://lightgbm.readthedocs.io/en/latest/Parameters-Tuning.html). Instead of L1 regularisation, try fitting a good boosting algorithm using these arguments only. Compare to best L1 reg performance.
